src/app/shell/shell.component.html

```diff
@@ -46,6 +46,16 @@
 
 					<mat-divider></mat-divider>
 
+					<a mat-list-item
+						*ngIf="organization && organization.organizationUrl"
+						href="#"
+						(click)="visitOrganizationUrl($event)"
```

Comment:
Curious why you didn't just set this is a href link and why you needed a click function

Reply:
forgot this was user input, this makes sense